### ISSUE TITLE
Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by juangelos** [Microsoft/vsts-task-lib#99]
Exact Repro Unknown: the bug occurs when using the ExtractFiles Task from vsts-tasks, cfr.
https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/ExtractFiles/extractfilestask.ts#L225

LOG: 
> 2016-07-04T20:41:40.8832784Z Extracting file: C:\vsobuild\_work\064c96c25\REL.xxxx\STAGING1\xxxx-rlsmgmt-STAGING1-20160701.4.zip
> 2016-07-04T20:41:40.8832784Z [command]7zip/7z.exe x -oc:\deploy\xxxx C:\vsobuild\_work\064c96c25\REL.xxxx\STAGING1\xxxx-rlsmgmt-STAGING1-20160701.4.zip
> 2016-07-04T20:41:40.8872795Z ##[error]Unhandled: Cannot read property 'toString' of null
> 2016-07-04T20:41:40.8882796Z ##[section]Finishing: Extract files 

Debugged it and traced the execution until line https://github.com/Microsoft/vsts-task-lib/blob/master/node/lib/toolrunner.ts#L366
which outputs into our log.

Problem seems to bee some weird condition causes the 7zip process not to have an stdout or stderr pipe at the .toString() lines, 379 & 380 

```csharp
        res.stdout = r.stdout.toString();
        res.stderr = r.stderr.toString();
```

**Comment by ericsciple** [comment]
@jasholl to confirm - i think this was fixed

**Comment by jasholl** [comment]
Yes, this is fixed with 610dbefbf97fb06c36c0238e69eb1aec2eecd1b2
